fiftyone/utils/sam.py

```
@@ -26,6 +26,26 @@
 logger = logging.getLogger(__name__)
 
 
+def _expand_sam_prompt_field(field_mapping):
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `_expand_sam_prompt_field`

Add return type annotation: `None`

(ANN202)